markdown/events/2022/bytesize-resources-to-learn-nextflow.md

```diff
@@ -23,3 +23,59 @@ It is our hope that these talks / videos will build an archive of training mater
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: The content has been edited for reader-friendliness**
+
+[0:01](https://www.youtube.com/watch?v=oO7rAp-QZOkt=1)
+Hello, everyone. My name is Franziska Bonath. I'm today's host of this talk, and with me is Sateesh Peri. And he is going to give a talk about resources to learn Nextflow. In the chat, he put a link for live slides. And from now, I give it over to Sateesh.
+
+[0:26](https://www.youtube.com/watch?v=oO7rAp-QZOkt=26)
+Thank you, Franziska. As mentioned, we have pasted a link to the live presentation that you can follow along. I'm going to be sharing my screen as well. So today here, I'm going to give you a walkthrough on all the resources that are currently available to get a good handle on the Nextflow workflow management system.
+
+[1:01](https://www.youtube.com/watch?v=oO7rAp-QZOkt=61)
+Lets dive straight into it. If you're here, some of the goals that you should be having are to learn a simple syntax for writing pipelines that enable you to reuse existing scripts and tools from past prototyping, and also to develop self-contained pipelines, manage versions, and to be able to reproduce any form of configuration on demand. One of the most useful resources was this blog post earlier in the year, that was very useful in bringing together all the useful links that are available. But since then, there have been a lot of developments, DSL2 is now default. We were wondering if they could give a more curated list of resources that are available to learn Nextflow through self learning. Before you dive into Nextflow and nf-core, some of the prerequisites that you need to master are scripting languages, bash programming, some experience with containers and software dependencies, and also quite some experience with version control systems. Not only for version controlling but also collaborating with open source communities such as nf-core.
+
+[2:34](https://www.youtube.com/watch?v=oO7rAp-QZOkt=154)
+Here are some links. There are enough links online to help you get up to speed. But here are some links that I have followed. Pretty much this presentation is going to be about how I have learned Nextflow over the past year. I'll be sharing links that I have found useful throughout my journey. These are some of the links here to get you started on all the prerequisites that you might require as you are about to get started learning Nextflow. Once you have those down, well, you're here because you want to learn Nextflow. Nextflow enables scalable and reproducible scientific workflows using software containers. It allows the adaptation of pipelines written in most common scripting languages, and its a fluent domain specific language, currently in its version two of revolution, simplifies implementation and deployment of complex parallel and reactive workflows on cloud and hybrid environments.
```

Comment:
```suggestion
Here are some links. There are enough links online to help you get up to speed. But here are some links that I have followed. Pretty much this presentation is going to be about how I have learned Nextflow over the past year. I'll be sharing links that I have found useful throughout my journey. These are some of the links here to get you started on all the prerequisites that you might require as you are about to get started learning Nextflow. Once you have those down, well, you're here because you want to learn Nextflow. Nextflow enables scalable and reproducible scientific workflows using software containers. It allows the adaptation of pipelines written in most common scripting languages, and its a fluent domain specific language, currently in its version two of evolution, simplifies implementation and deployment of complex parallel and reactive workflows on cloud and hybrid environments.
```